libraries/fc/src/crypto/elliptic_r1.cpp

```diff
@@ -16,7 +16,7 @@ namespace fc { namespace crypto { namespace r1 {
           public_key_impl()
           :_key(nullptr)
           {
-          static int init = init_openssl();
+            init_openssl();
```

Comment:
The static is used so that init_openssl() is only called once.

Reply:
Got it!